content/en/docs/tasks/job/indexed-parallel-processing-static.md

```diff
@@ -26,14 +26,14 @@ expose the index in the `JOB_COMPLETION_INDEX` environment variable.
 
 Here is an overview of the steps in this example:
 
-1. **Create an image that can read the pod index**. You might modify the worker
-   program or add a script wrapper.
-2. **Start an Indexed Job**. The downward API allows you to pass the annotation
-   as an environment variable or file to the container.
+1. **Define a Job manifest using indexed completion**.
+   The downward API allows you to pass the pod index annotation as an
+   environment variable or file to the container.
+2. **Start an `Indexed` Job based on that manifest**.
```

Comment:
👋 @sftim ,

Is there a reason to use bold and code formatting in the instructions?
An indexed Job is a `Job`?

Reply:
The formatting here is as per the original.

Reply:
An Indexed Job is a Job with `.spec.completionMode=Indexed`. Bold could be omitted. I just think it makes it easier to get the gist of the steps.